crates/oxc_cli/src/options.rs

```diff
@@ -0,0 +1,24 @@
+use std::path::PathBuf;
+
+use clap::ArgMatches;
+
+pub struct CliOptions {
+    pub quiet: bool,
+    pub path: PathBuf,
+}
+
+impl<'a> TryFrom<&'a ArgMatches> for CliOptions {
+    type Error = &'a str;
+
+    fn try_from(matches: &ArgMatches) -> Result<Self, Self::Error> {
+        let path = matches.get_one::<PathBuf>("path").unwrap();
+        if path.canonicalize().is_err() {
+            return Err("err");
+        }
+
+        Ok(Self {
+            quiet: matches.get_one::<String>("quiet").is_some(),
```

Comment:
Did you consider using `<bool>` instead of `<String>`?

Reply:
I did yeah, but that didnt seem to work. I havent tried to make it its own command yet though, ill get to that today 🙂

Reply:
Actually, in combination with `.action(clap::ArgAction::SetTrue)` `matches.get_one::<bool>` does work!